compose/service.py

```diff
@@ -157,7 +157,12 @@ def scale(self, desired_num):
         - starts containers until there are at least `desired_num` running
         - removes all stopped containers
         """
-        if not self.can_be_scaled():
+        if self.custom_name() and desired_num > 1:
+            log.warn('Service %s specifies a custom container name. The Docker daemon '
```

Comment:
'Service %s is attempting to use a container named  % self.name. Docker requires each container have a unique name.'